### ISSUE TITLE
[Feature Request] IMIE and GSM details in device info and Binding with existing apps

### ISSUE THREAD

**Issue by shariqmalik** [qH0sT/AndroSpy#23]
Hey, first of all, thanks a lot for such a great thing.
I would like to request some features like binding with the existing app and IMEI and other information of Mobile in the device information menu as well.

Thanks

**Comment by kira25kira** [comment]
Yes, I agree with the brother @shariqmalik   and I hope it was possible run live screen With out accepting the phone
and Thanks for all the efforts

**Comment by qH0sT** [comment]
> Yes, I agree with the brother @shariqmalik and I hope it was possible run live screen With out accepting the phone
> and Thanks for all the efforts

I'm working on this; GSM Info and IMEI fetch. On the other side in my opinion your second suggest is impossible because MediaProjection require accept but may be there would be any hack trick.

Stay tuned to the project, I'll post the new update soon.

you are welcome :)

**Comment by carlossaldanha** [comment]
@qH0sT Do you have the possibility to install apps remotely on android without user interaction, for a client update?

**Comment by qH0sT** [comment]
@shariqmalik @kira25kira 

I have add feature : "Detailed Infos". Noon (12.12.2020) I will try the update on github.

![0](https://user-images.githubusercontent.com/45147475/101833176-a281d080-3b49-11eb-8e6d-1729f026a8b9.PNG)

@carlossaldanha I don't know, may be..

**Comment by shariqmalik** [comment]
@qH0sT what about binding it with any existing app? Can you make it possible?

**Comment by qH0sT** [comment]
> @qH0sT what about binding it with any existing app? Can you make it possible?

@shariqmalik 
I can actually make this; read base64 string of another .apk and write it into string.xml as resource string of our trojan. When trojan run, extarct base64 encoded .apk into the any directory then run it. May be later i can make this.

I have released new Update, check it.

**Comment by shariqmalik** [comment]
Thanks @qH0sT checking it. ✌🏻

**Comment by shariqmalik** [comment]
@qH0sT  I tested it but on SDK 29 Android 10 it's not getting IMEI and Device Serial.

![2020-12-11_16-38](https://user-images.githubusercontent.com/19536819/101899315-70b34d00-3bcf-11eb-859e-8dc36421601b.png)

**Comment by qH0sT** [comment]
> @qH0sT I tested it but on SDK 29 Android 10 it's not getting IMEI and Device Serial.
> 
> *Tested on 2 Android 10 devices.
> 
> ![2020-12-11_16-38](https://user-images.githubusercontent.com/19536819/101899315-70b34d00-3bcf-11eb-859e-8dc36421601b.png)

Android Q has restricted to access for both IMEI and serial no. It is available only for platform and apps with special carrier permission. Also the permission READ_PRIVILEGED_PHONE_STATE is not available for non platform apps.

**Comment by shariqmalik** [comment]
Oh, so that means we don't have any way to get the IMEI number of Android 10 and above?

**Comment by qH0sT** [comment]
> Oh, so that means we don't have any way to get the IMEI number of Android 10 and above?

yes, we don't have any way to IMEI because of security issue.
